platform/qt/src/qmapboxgl.cpp

```diff
@@ -1085,11 +1346,42 @@ void QMapboxGL::render()
 }
 #endif
 
+/*!
+    Informs the map that the network connection was established, causing
+    all timeouted network requests to retry immediately.
```

Comment:
s/was established/has been established/
s/timeouted network requests/network requests that previously timed out/
s/to retry/to be retried/